<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="notify">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>أضافة زبون</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_Customer">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>أضافة</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="target_date">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="clothes_model">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>الموديل</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص نسائي بدون اكمام</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص نسائي رسمي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص نسائي باكمام نفوشة</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>بنطال رياضي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>بنطال بجيوب جانبية</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص رجالي رسمي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص رجالي كاجول</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="clothes_categories">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>فئة الملابس </string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>تنورة طويلة</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>تنورة قصيرة</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>بنطال</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>قميص</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>تي-شيرت</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ثوب</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>بذلة رسمية</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                 <string> ادخل مقاسات الجسم بالسنتمتر</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="shoulder_size">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="placeholderText">
                 <string>  الكتف للكتف</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="waist_size">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>  الخصر</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="hip_size">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string> الارجل</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fabric_type">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="placeholderText">
              <string> نوع القماش</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="customer_name">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="placeholderText">
              <string> اسم الزبون</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="customertable">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <column>
           <property name="text">
            <string>اسم الزبون</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>نوع القماش</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>قياس الكتف للكتف</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>قياس الخصر</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>قياس الارجل</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>فئة الملابس </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>الموديل</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>وقت الانتهاء</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>بحث عن قياسات زبون</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="customer_n">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string> اسم الزبون</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="customer_search">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string> ابحث</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="measurementstable">
          <column>
           <property name="text">
            <string>قياس الكتف للكتف</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>قياس الخصر</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>قياس الارجل</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>التنبيهات</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableWidget" name="notifytable">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <column>
           <property name="text">
            <string>موعد التسليم</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
